_targets.R

```diff
@@ -1,59 +1,9 @@
 library(targets)
-source("1_fetch/src/get_nwis_data.R")
-source("2_process/src/process_and_style.R")
-source("3_visualize/src/plot_timeseries.R")
+library(tidyverse)
 
-options(tidyverse.quiet = TRUE)
-tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
```

Comment:
You need to leave the `tar_option_set()` line for "dataRetrieval" because otherwise `targets` has no way of knowing about the functions you call from `dataRetrieval`. It is working right now because the pipeline is skipping all of those targets and it hasn't tried to use any `dataRetrieval` functions since you deleted this.